ports/cppslippi/portfile.cmake

```
@@ -0,0 +1,21 @@
+vcpkg_from_sourceforge(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO            cppslippi
+    FILENAME        "CppSlippi-${VERSION}.zip"
+    SHA512          c6daebd5c28807d0addf77c4e3974a9811cc3d054dd57c2d19f0977a3d87c0718cd99c301b40d172b54512dcbad259477e7d54977558d70ef94922146d1680ff
+    NO_REMOVE_ONE_LEVEL
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DBUILD_TESTING=False
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/CppSlippi)

```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/share/CppSlippi)
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/CppSlippi)
```
Since `CppSlippi-config.cmake` was installed in _lib/share_ folder, so need to handle it too.